CONTRIBUTORS.md

```diff
@@ -3,10 +3,10 @@ Chapel Contributors
 
 The following people have contributed to Chapel releases:
 
-Contributors to the Chapel 2.0 release
+Contributors to the Chapel 2.1 release
 --------------------------------------
 * Jade Abraham, [HPE]
-* Paul Cassella, [HPE]
+* Dan Bonachea, [Lawrence Berkeley National Laboratory, formerly UC Berkeley]
```

Comment:
@bradcray Thanks for adding me!

I think it's fine to drop the UC Berkeley affiliation : even when I was a student at UC Berkeley working on GASNet for Chapel I was actually being paid by a grant through LBNL. Also both are operated by UC Regents anyhow so the distinction is marginal.

However on a presentational note, I believe this is intended to be a markdown link, and as currently written the link is broken. 

Simple fix to both problems:

```suggestion
* Dan Bonachea, [Lawrence Berkeley National Laboratory]
```

Or if you wish to also link my home page:

```suggestion
* [Dan Bonachea](https://go.lbl.gov/dan-bonachea), [Lawrence Berkeley National Laboratory]
```